lib/actions/app.coffee

```diff
@@ -169,7 +191,15 @@ exports.associate =
 					return callback(new Error("Invalid application: #{params.name}"))
 
 				message = "Are you sure you want to associate #{currentDirectory} with #{params.name}?"
-				visuals.patterns.confirm(options.yes, message, callback)
+				if options.yes
+					return callback(null, true)
+				else
+					form.ask
+						message: message
+						name: 'confirm'
+						type: 'confirm'
+						default: false
+			    .nodeify(callback)
```

Comment:
Is this `.nodeify()` intended to match `form.ask()`? If so, there are indentation issues here.